CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [27.1.0]
+### Changed
+- Now the `KeyringController` supports the `QRKeyring` from `@keystonehq/metamask-airgapped-keyring`. Developers can enable the import of accounts from a QR hardware wallet. ([#685](https://github.com/MetaMask/controllers/pull/685))
```

Comment:
Maybe this should go under Added?

Reply:
I was between `Changed` and `Added` since it changes the Keyring but adds new method. I'll follow your recommendation [e25ad64](https://github.com/MetaMask/controllers/pull/755/commits/e25ad647f5c63144c9103e608c3ba9ee2343a35e)

Reply:
Now I see it makes reference to the feature itself